README.md

```diff
@@ -248,8 +248,13 @@ can be configured via the [config.json](#configuration-files).
 # Driver model
 
 Gaia hub drivers are fairly simple. The biggest requirement is the ability
-to fulfill the _write-to/read-from_ URL guarantee. As currently implemented
-a gaia hub driver must implement the following two functions:
+to fulfill the _write-to/read-from_ URL guarantee. 
+
+A driver can expect that each of its functions will be mutexed by a path. 
```

Comment:
Grammar -- I don't think mutex is a verb?

"A driver can expect that two POSTs to the same path will be mutually exclusive.  No POSTs to the same path will be concurrent."

Reply:
Fixed